Jenkinsfile

```diff
@@ -183,6 +183,88 @@ pipeline {
                 }
             }
         }
+
+        stage('Build on Linux no nccl xgboost') {
+            agent {
+                label "gpu && nvidia-docker && !mr-dl16"
+            }
+
+            steps {
+                dumpInfo 'Linux Build Info'
+                // Do checkout
+                retryWithTimeout(100 /* seconds */, 3 /* retries */) {
+                    deleteDir()
+                    checkout([
+                            $class                           : 'GitSCM',
+                            branches                         : scm.branches,
+                            doGenerateSubmoduleConfigurations: false,
+                            extensions                       : scm.extensions + [[$class: 'SubmoduleOption', disableSubmodules: true, recursiveSubmodules: false, reference: '', trackingSubmodules: false, shallow: true]],
+                            submoduleCfg                     : [],
+                            userRemoteConfigs                : scm.userRemoteConfigs])
+                }
+
+                script {
+                    CONTAINER_NAME = "h2o4gpu${SAFE_CHANGE_ID}-${env.BUILD_ID}"
+                    // Get source code
+                    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: "awsArtifactsUploader"]]) {
+                        try {
+                            sh """
+                                    nvidia-docker build -t opsh2oai/h2o4gpu-build -f Dockerfile-build .
+                                    nvidia-docker run --init --rm --name ${CONTAINER_NAME} -d -t -u `id -u`:`id -g` -v /home/0xdiag/h2o4gpu/data:/data -v /home/0xdiag/h2o4gpu/open_data:/open_data -w `pwd` -v `pwd`:`pwd`:rw --entrypoint=bash opsh2oai/h2o4gpu-build
+                                    nvidia-docker exec ${CONTAINER_NAME} rm -rf data
```

Comment:
@abal5 could it be the `exec`- part replaced by a script? Each stage doing the same, seems like it could be easily squashed to a single command.